### DISCUSSION TITLE
Going Open Source

### DISCUSSION THREAD

**Discussion by difcsi** [difcsi/e5vosdosite#201] [Ideas]
Jót tenne a projektnek ha open source lenne: nekünk is jó portfólióba, eötvös diáknak is jó mert nagyobb kedve lesz belenyúlni ha látja a szoftvert

Ezügyben elkezdtem pár markdown doksit írni, majd felcommitolom.

Majd repo secret resetet kell csinálni előtte mert most benne van az e5vos.dev api keytől kezdve minden is

**Comment by volubepossum** [comment]
ki commitolt api keyt
amúgy egyet értek

**Comment by FCsacsa** [comment]
Esetleg egy részletesebb readme is segítene hogy mások is tudjanak hozzányúlni a kódhoz.
